kaos/parser.py

```diff
@@ -3,9 +3,15 @@
 from collections import namedtuple
 from kaos.models import DB, OrbitRecords, SatelliteInfo, OrbitSegments
 from sqlalchemy import or_, and_
+import math
 
 OrbitPoint = namedtuple('OrbitPoint', 'time, pos, vel')
 
+def get_length_q(sat_position):
```

Comment:
I think this is general enough that it might warrant being moved to utils/vectors.py. Thoughts from others?

Reply:
Even better!

Reply:
any suggestions on moving get_length_q() to utils/vectors.py ?

Reply:
of course